pkg/kube/apis/extendedjob/v1alpha1/types.go

```diff
@@ -1,16 +1,26 @@
 package v1alpha1
 
 import (
+	"fmt"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
+
+	"code.cloudfoundry.org/cf-operator/pkg/kube/apis"
 )
 
 // This file is safe to edit
 // It's used as input for the Kube code generator
 // Run "make generate" after modifying this file
 
+var (
+	// LabelReferencedJobName is the name key for dependent job
+	LabelReferencedJobName = fmt.Sprintf("%s/referenced-job-name", apis.GroupName)
+	// LabelReferencedSecretName is the name key for dependant's referenced secret
+	LabelReferencedSecretName = fmt.Sprintf("%s/referenced-secret-name", apis.GroupName)
```

Comment:
We shouldn't need this anymore

Reply:
I thought we use it to fetch the latest version when update versioned secret in reconciler.